### PR TITLE
Product of Dirichlet series

### PR DESCRIPTION
**Pull request by VictorBlanchi**
In this PR, I have shown that the Dirichlet series of the Dirichlet product is the product of the Dirichlet series for species of types and species of types in subuniverse 

(it is an old branch, and I have just notice I didn't merge it yet)

### CODE REVIEW

File: src/species/products-dirichlet-series-species-of-types-in-subuniverses.lagda.md [hunk 1]
```diff
@@ -56,3 +69,111 @@ module _
       ( T)
       ( X)
 ```
+
+## Properties
+
+### The Dirichlet series associated to the Dirichlet product of `S` and `T` is equal to the product of theirs Dirichlet series
```
Comment (fredrik-bakke) [inline src/species/products-dirichlet-series-species-of-types-in-subuniverses.lagda.md:75]: product of their~s~*

**Review by fredrik-bakke — APPROVED** [review]
Looks very good! 😄

**Comment by fredrik-bakke** [comment]
By the way, are there any good references on Dirichlet series, or species more generally, that can be added?

**Comment by EgbertRijke** [comment]
Not sure if we still have Victor's attention, so I'm just merging this.

**Pull request merged**